packages/cdk/target_tracking_scaling_policy/config-PROD.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
could you avoid having the duplicated configuration by having a generic -STAGE file which the put-scaling-policy script copies, uses sed to set the correct stage, then runs the cli command and deletes?

Reply:
good idea!